pkgs/development/libraries/gstreamer/rs/default.nix

```diff
@@ -101,7 +102,7 @@ let
       "reqwest" # tests hang on darwin
       "threadshare" # tests cannot bind to localhost on darwin
       "webp" # not supported on darwin (upstream crate issue)
-    ] ++ lib.optionals (stdenv.isDarwin && !stdenv.isAarch64) [
+    ] ++ lib.optionals ((stdenv.isDarwin && !stdenv.isAarch64) || !gtkSupport) [
```

Comment:
Can you just make this `!gtkSupport` and then adjust the default value for `gtkSupport` to be `gst-plugins-base.glEnabled`?